Main.py

```diff
@@ -115,26 +147,31 @@
     max_torrents = args.max
 if args.shows:
     show_list = args.show
+if args.api:
+    api_root = args.api
 
 file = open('last_torrent', 'r')
 last_seen_torrent = int(file.readline())
 file.close()
+logger.debug("last_seen_torrent: %d" % last_seen_torrent)
 
 # get JSON from EZTV
 request = []
+request_string_base = api_root + '?limit=' + str(max_torrents) + '&page='
```

Comment:
Instead of using string concatenation, you could pass a dict with your params to the `requests` library which would be clearer.

http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

Reply:
That makes a lot of sense, thanks! 🦊